tests/test_extensions.py

```diff
@@ -573,8 +573,6 @@ def testRE(self):
 class TestTOC(TestCaseWithAssertStartsWith):
     """ Test TOC Extension. """
 
-    unittest.TestCase.maxDiff = None
```

Comment:
Please don't remove this. I want to see a complete diff of any test failure when debugging a problem.

Reply:
It is what I added and it does not exist in the previous commit.

Reply:
Setting [maxDiff][1] to `None` causes it to display the complete diff (unlimited length). By removing the line, `maxDiff` falls back to the default of 80*8 characters. Therefore, please restore the line so we have complete diffs.

[1]: https://docs.python.org/2/library/unittest.html#unittest.TestCase.maxDiff